saleor/plugins/webhook/shipping.py

```diff
@@ -127,18 +128,14 @@ def get_excluded_shipping_methods_from_response(
     excluded_methods = []
     for method_data in response_data.get("excluded_methods", []):
         try:
-            raw_id = method_data["id"]
-            typename, _id = graphene.Node.from_global_id(raw_id)
-            if typename == APP_ID_PREFIX:
-                method_id = raw_id
-            elif typename == "ShippingMethod":
-                method_id = _id
-            else:
+            type_name, method_id = from_global_id_or_error(method_data["id"])
+            if type_name not in (APP_ID_PREFIX, "ShippingMethod"):
```

Comment:
Do we have a `ShippingMethod` constant like `APP_ID_PREFIX`? It would be good to have it imho

Reply:
it is used also in `saleor/graphql/core/utils/__init__.py` in `from_global_id_or_error` function. Thats why didn't treat it as ShippingMethod constant and left it in utils as it were in original PR for 3.0

Reply:
It refers to the `ShippingMethod` type in API. I wouldn't add any constant for that, if anything we could import the type and do `str(ShippingMethod)` or something like this.

Reply:
changed to use `str(ShippingMethod)`